src/util.js

```diff
@@ -37,10 +37,8 @@ export function buildIdentifierQuery(patron, idents) {
 }
 
 export function buildRequestQuery(requesterId, servicePointId) {
-  const servicePointClause = servicePointId ? `pickupServicePointId=${servicePointId} and` : '';
-  return `(requesterId==${requesterId} and
-    ${servicePointClause}
-    status=="Open - Awaiting pickup")`;
+  const servicePointClause = servicePointId ? `pickupServicePointId=${servicePointId} and ` : '';
+  return `(requesterId==${requesterId} and ${servicePointClause}status=="Open - Awaiting pickup")`;
```

Comment:
Could you please create const with open status
```
export const OPEN_REQUEST_STATUSES = {
  OPEN_NOT_YET_FILLED: 'Open - Not yet filled',
  OPEN_AWAITING_PICKUP: 'Open - Awaiting pickup',
  OPEN_IN_TRANSIT: 'Open - In transit',
  OPEN_AWAITING_DELIVERY: 'Open - Awaiting delivery',
};
```
 in ui-checkout/src/constants.js and use `"Open - Awaiting pickup" `from const?